src/content/self-hosted/install/deployment.mdx

```diff
@@ -29,25 +32,6 @@ Run the following command to obtain your cluster's API server endpoint:
 kubectl config view --minify | grep server
 ```
 
-### Domain
-
-You'll need sufficient access to a [subdomain](self-hosted/administration/configuration.mdx#subdomain) to add a wildcard DNS record, such as dev.example.com.
-By default, all endpoints created by Okteto for your development environments will be exposed on the wildcard subdomain you choose.
-
-```yaml
-subdomain: "dev.example.com"
-```
-
-### License
-
-Okteto is free for small teams.  You get all the features of [Okteto](self-hosted.mdx) for up to 3 users.
```

Comment:
Let's mention that it's optional, but encourage to get a license by starting the free trial?